canonical-docs/conjure-up-docs
# Entrypoint for talisker gevent workers

Fixed https://github.com/ubuntudesign/base-squad/issues/105

QA
--

``` bash
./run
```

Go to http://localhost:8204, check it works. Now:

``` bash
docker build --tag conjure-up-docs --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 conjure-up-docs
```

Go to http://localhost:8099, check that also works.

Also, check the `X-VCS-Revision` shows correctly:

``` bash
$ curl -Is localhost:8099 | grep VCS
X-VCS-Revision: d5e6b777c7b29b057b2aa104e7ca0778ec197068
```
